### ISSUE TITLE
SBSDKScannerViewController.cameraControlsHidden hides realtime polygon

### ISSUE THREAD

**Issue by irvingruan** [doo/Scanbot-SDK-Examples-iOS#5]
SBSDKScannerViewController's ```cameraControlsHidden``` property also hides the live polygon and custom HUD that can get set via the delegate method, ```- (UIView *)viewForDetectionStatus:(SBSDKDocumentDetectionStatus)status forScannerController:(SBSDKScannerViewController *)controller```, even though the documentation for the former states the shutter button as the example.

I would like to propose having two exposed properties in SBSDKScannerViewController's public API:

```
@property (nonatomic, readwrite, assign, getter=isDetectionStatusViewHidden) BOOL detectionStatusVideHidden
@property (nonatomic, readwrite, assign, getter=isShutterButtonHidden) BOOL shutterButtonHidden
```

That way, developer can decide which views they want to customize themselves.

**Comment by Constantine-Fry** [comment]
Hi, @irvingruan.

Unfortunately, the developer who is working on ScanbotSDK is on vacation. Is it something really important for you or it can wait for 2 weeks?

**Comment by irvingruan** [comment]
@Constantine-Fry: It can wait. Thanks!

**Comment by sebastato** [comment]
Fixed in 1.0.5.
